consensus/qbft/src/integration-test/java/org/hyperledger/besu/consensus/qbft/test/LocalNodeNotProposerTest.java

```diff
@@ -92,6 +95,30 @@ public void preparesReceivedFromNonProposerIsValid() {
     assertThat(context.getCurrentChainHeight()).isEqualTo(1);
   }
 
+  @Test
+  public void prepareFromProposerIsIgnored() {
```

Comment:
should this test be removed? I.e. we ACCEPT (not ignore) the prepare from the proposer now.

Reply:
Yes it should be. It doesn't make sense now.